CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ STRING(TOUPPER          "${CMAKE_PROJECT_NAME}" PROJECT_NAME_UCASE)
 # Updated by maintenance tasks
 SET(PROJECT_YEAR 2015)
 
-SET(PROJECT_AUTHOR      "LMMS Developers")
+SET(PROJECT_AUTHOR      "LMMS developers")
```

Comment:
Why lowercase here?  Is this necessary change as part of this PR or was it something that was never reverted after testing?

Reply:
It's the latter. I will capitalize it.